R/calc.R

```diff
@@ -46,10 +46,10 @@
 #'
 #' test.results <- calc(test.data,
 #'                      what = c("kotelchuck", "fetal_pres"),
-#'                      "chi_year == 2016 & chi_sex %in% c('Male', 'Female')",
+#'                      chi_year == 2016 & chi_sex %in% c('Male', 'Female'),
```

Comment:
no 'where'

Reply:
You mean like making the command be `where = chi_year == 2016 & chi_sex %in% c('Male', 'Female')`

Reply:
yes
I'd also love it if it could deal with the argument having quotes or not having quotes, but I bet that will make you batty!
`where = "chi_year == 2016 & chi_sex %in% c('Male', 'Female')"`